### ISSUE TITLE
1.3.7 available via PyPi

### ISSUE THREAD

**Issue by petrklus** [SparkPost/python-sparkpost#196]
Hi, 

Is there a reason why the latest release (1.3.7) is not available via PyPI?

**Comment by hugoseabra** [comment]
Yes @petrklus . I've recently contacted the Sparkpost support and they told me that this project is no longer supported (and maybe they are looking for good guessers to realize that).

They allowed me to maintain the project in a forked repository. I'll try to keep it updated by myself, unfortunately.
